File: app/src/main/java/com/example/bervianto/ecceg_rsa_app/ecc/Point.java
package com.example.bervianto.ecceg_rsa_app.ecc;

import java.math.BigInteger;

public class Point {
	public BigInteger x, y; // x = absis, y = ordinat
	public boolean infinity; // titik O, elemen identitas

	public Point() {
		x = BigInteger.ZERO;
		y = BigInteger.ZERO;
		infinity = false;
	}

	public String toString(){
		return "(" + x + ", " + y + ")";
	}
}

File: app/src/main/java/id/my/berviantoleo/ecceg_rsa_app/lib/ecc/Point.kt
package id.my.berviantoleo.ecceg_rsa_app.lib.ecc

import java.math.BigInteger

class Point {
    var x: BigInteger = BigInteger.ZERO
    var y: BigInteger = BigInteger.ZERO // x = absis, y = ordinat
    var infinity: Boolean = false // titik O, elemen identitas

    override fun toString(): String {
        return if (infinity) "Point(O)" else "($x, $y)"
    }
}
